ebd-all-minilm/main.py

```diff
@@ -9,6 +11,15 @@
 
 load_dotenv()
 
+MODEL_PATH = f"./ml/models/{os.getenv('MODEL_ID', 'all-MiniLM-L6-v2')}"
```

Comment:
actually looks like this causes docker builds to fail (both for this PR and in `cht-llama-cpp`)...

```suggestion
MODEL_PATH = os.getenv('MODEL_ID', 'all-MiniLM-L6-v2')
```

wdyt @biswaroop1547?

Reply:
also if `MODEL_PATH` is meant to be a directory (not a file), then should call it `MODEL_DIR` instead @nsosio :)

Reply:
> Also the cht-llama-cpp version has  .gguf extension. Is that needed here too?

based on [this](https://github.com/premAI-io/prem-registry/pull/98/files#diff-1ae74df7e6f42efffe1c1599958b1f71c5568a9d9b55c6391380d3af317ceddeR24) I think it won't be needed.

Reply:
@casperdcl done :) @biswaroop1547 yes for this one we have a folder of files so no extension is needed